### DISCUSSION TITLE
Feature Request : Colored tags for better organization

### DISCUSSION THREAD

**Discussion by ConstantMan** [standardnotes/forum#2644] [Suggestions]
I really like the fact that I can have an open source and privacy respecting notes app. But it lacks some basic features that you can see how nice they would look on your app when instead of only showing protected /pinned it would show the different tags with a different color each.

![image](https://user-images.githubusercontent.com/44656000/124099795-bbcd6a00-da12-11eb-87b4-9503497ce773.png)

It's really useful to have the ability to color filter each of the tags especially if you have lots of them. And even if you don't want to add color I don't see why they should appear on the app dashboard, since the only way to categorize the notes for now is to choose each different category. 

I'm on the free plan but still it's a very basic feature even for simple note taking apps like Samsung/Huawei/etc default app. And very useful when you have a lot of notes to sync between desktop and mobile and you want to immediately see what notes you have in each category. Also I don't think there is a problem regarding encryption since you can already have different tags and even have different themes with extensions.

**Comment by JaspalSuri** [comment]
Hi @KonstantMan, thanks for the suggestion!

As for viewing tags in the notes list under each note, I'm afraid that there is a technical limitation that prevents us adding support for tags in the notes list on the mobile apps, but we plan to re-implement it as soon as it's possible. 🙂

**Comment by ConstantMan** [comment]
Also another idea would be to have a different color in each note depending on the tag on the notes list

**Comment by jordan-bravo** [comment]
+1 on requesting the feature to color code notes (or color codes tags on notes).

**Comment by myreli** [comment]
While colours aren't supported, now we can customize icons:

<img width="471" alt="image" src="https://user-images.githubusercontent.com/17554234/218858835-f43402cc-1d87-4a2d-9050-6c8b5bb302c4.png">

**Reply by jordan-bravo** [reply]
A couple questions on this feature:

 -  How do you configure this?
 -  Is this for mobile, desktop, both?

**Reply by myreli** [reply]
Hi @jordan-bravo! 

- On web you can either right-click a tag, or click on a tag and then press the ... button that appears when it's selected. On mobile you can long-press or also select and then press the ... button. Let me know if you find it, or I can record a quick gif. 
- Both! 🙂 The screenshot is on desktop.
